doc/rst/basic_usage.rst

```diff
@@ -31,13 +31,23 @@ Endpoint Type Tables
 The endpoint type tables are arrays of type ``XUD_EpType`` and are used to inform ``lib_xud``
 about the endpoints in use.  This information is used to indicate the transfer-type of each endpoint
 (bulk, control, isochronous or interrupt) as well as whether the endpoint wishes to be informed
-about bus-resets (see `Status Reporting`_).
+about bus-resets (see `Status Reporting`_). Two tables are requires, one for IN and one for OUT
+endpoints.
 
-.. note::
+Suitable values are provided in the ``XUD_EpTransferType`` `enum`:
+
+    * ``XUD_EPTYPE_ISO``: Isochronous endpoint
+    * ``XUD_EPTYPE_INT``: Interrupt endpoint
+    * ``XUD_EPTYPE_BUL``: Bulk endpoint
+    * ``XUD_EPTYPE_CTL``: Control endpoint
+    * ``XUD_EPTYPE_DIS``: Disabled endpoint
 
-    Endpoints can also be marked as disabled.
+Endpoint 0 uses index 0 of both the endpoint type table. The address of other endpoints corresponds
```

Comment:
Suggest a slight reword: IN endpoint N will use index N of the input-endpoint-table, OUT endpoint 0x8N will use index N of the output-endpoint-table. Endpoint 0 must exist in both tables.